server/chat-commands/info.ts

```diff
@@ -1886,7 +1889,7 @@ export const commands: ChatCommands = {
 				`- /rules <em>rules link</em>: set the room rules link seen when using /rules`,
 				`- /roommod, /roomdriver <em>username</em>: appoint a room moderator/driver`,
 				`- /roomdemod, /roomdedriver <em>username</em>: remove a room moderator/driver`,
-				`- /roomdeauth <em>username</em>: remove all room auth from a user`,
+				`- /roomdeauth <em>username</em>: remove all room.settings.auth from a user`,
```

Comment:
```suggestion
				`- /roomdeauth <em>username</em>: remove all room auth from a user`,
```

Let me guess, you used a regex so searching for `room.auth` meant "any character between `room` and `auth`".

Reply:
correct! i thought i got a lot of these reverted but apparently i missed this one